auth/src/main/java/com/firebase/ui/auth/IdpResponse.java

```diff
@@ -34,22 +34,25 @@
 public class IdpResponse implements Parcelable {
     private final String mProviderId;
     private final String mEmail;
+    private final String mPhoneNumber;
```

Comment:
Oh, K so we do want the dev to get the phone number from `IdpResponse`. Cool beans! 😄